partridge/writers.py

```diff
@@ -35,15 +36,22 @@ def write_feed_dangerously(feed, outpath, nodes=None):
     your own risk.
     """
     nodes = DEFAULT_NODES if nodes is None else nodes
+
     try:
         tmpdir = tempfile.mkdtemp()
 
-        for node in nodes:
+        def write_node(node):
             df = feed.get(node)
             if not df.empty:
                 path = os.path.join(tmpdir, node)
                 df.to_csv(path, index=False)
 
+        try:
+            pool = ThreadPool(len(nodes))
```

Comment:
Should this be outside the `try` so that we don't `terminate()` a non-existent pool if `ThreadPool()` fails for some reason?

Reply:
Yep.

Reply:
Done.